src/test/java/org/folio/rest/impl/BankingInformationTest.java

```diff
@@ -0,0 +1,75 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.jaxrs.model.BankingInformation;
+import org.folio.rest.jaxrs.model.BankingInformationCollection;
+import org.folio.rest.utils.TestEntities;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.UUID;
+
+import static org.folio.rest.utils.TenantApiTestUtil.prepareTenant;
+import static org.folio.rest.utils.TenantApiTestUtil.purge;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class BankingInformationTest extends TestBase {
```

Comment:
Doesn't `EntitiesCrudTest` covers all CRUD operations if you add new entity to `TestEntities` enum?